nengo_gui/examples/tutorial/22-spa-sequence-controlled.py

```diff
@@ -32,8 +38,6 @@
         'dot(memory, D) --> memory=E',
         'dot(memory, E) --> memory=vision',
         )
-        
```

Comment:
Hmm...  how do you feel about tossing in an extra action that is ```'0.5 --> memory=vision',```?  That seems to avoid a few cases for me where it gets stuck not doing anything....